kernel_crawler/__init__.py

```diff
@@ -44,16 +47,25 @@ def to_driverkit_config(d, res):
 
     return dk_configs
 
-def crawl_kernels(distro, version, arch, to_driverkit):
+def crawl_kernels(distro, version, arch, image, to_driverkit):
```

Comment:
Super nitpick: what about calling `images`?
```suggestion
def crawl_kernels(distro, version, arch, images, to_driverkit):
```